db/migrate/20210312004640_add_record_to_access_log.rb

```diff
@@ -0,0 +1,20 @@
+class AddRecordToAccessLog < ActiveRecord::Migration[6.0]
+  def up
+    # add the new columns (does this make indices?)
+    add_reference :access_logs, :record, polymorphic: true
+    # set the new columns
+    update "UPDATE access_logs SET record_type = 'Client', record_id = client_id"
+    # make the new columns non-nullable
+    change_column_null :access_logs, :record_id, false
+    change_column_null :access_logs, :record_type, false
+    # make the old column nullable
+    change_column_null :access_logs, :client_id, true
+  end
+
+  def down
+    # remove the old columns
+    remove_reference :access_logs, :record, polymorphic: true
+    # we can't reset client_id to null false, it's not safe
+  end
+end
+
```

Comment:
The previous migration:
https://github.com/codeforamerica/vita-min/pull/901/files#diff-43b5d75f9dc345eb9e4109db7ef08d244489f81ed86238e6dedadd30dd6f3014R1-R15

Once this is deployed, we can follow up with a migration that removes the client_id column